src/mozanalysis/bq.py

```diff
@@ -63,11 +64,11 @@ def run_query(self, sql, results_table=None):
                     )
                 ),
             ).result()
-            print("Saved into", results_table)
+            logging.info("Saved into", results_table)
```

Comment:
Is there a reason to use `logging.info()` instead of instantiating a logger with getLogger and then using `logger.info()`?

Reply:
yeah, getLogger probably makes more sense here